selfdrive/controls/controlsd.py

```diff
@@ -86,8 +87,11 @@ def __init__(self, sm=None, pm=None, can_sock=None, CI=None):
       ignore = ['testJoystick']
       if SIMULATION:
         ignore += ['driverCameraState', 'managerState']
-      if self.params.get_bool('WideCameraOnly'):
-        ignore += ['roadCameraState']
+        # block until get the available streams
+        available_streams = VisionIpcClient.available_streams("camerad", block=True)
```

Comment:
hmm, this can't block too long or else controlsd won't run in case of camera malfunctions. maybe we can fit it in during the initializing state? https://github.com/commaai/openpilot/blob/a86730a2c601188c480285c45999efa00c3b3d4a/selfdrive/controls/controlsd.py#L434